website/docs/language/expressions/references.mdx

```diff
@@ -121,10 +121,11 @@ to mark the reference as for a data resource.
 * `path.module` is the filesystem path of the module where the expression
   is placed.
 * `path.root` is the filesystem path of the root module of the configuration.
-* `path.cwd` is the filesystem path of the current working directory. In
-  normal use of Terraform this is the same as `path.root`, but some advanced
-  uses of Terraform run it from a directory other than the root module
-  directory, causing these paths to be different.
+* `path.cwd` is the filesystem path of the current working directory. This is
```

Comment:
There's a bit of an odd interaction with modern versions of Terraform and the `-chdir` argument which makes this sentence technically incorrect. Here's my understanding of the current logic, worded clumsily, hoping that the technical writing team can help 🙏 

```suggestion
* `path.cwd` is the filesystem path of the original working directory where you ran Terraform from, before any `-chdir` argument is applied. This is
```